docs/source/conf.py

```diff
@@ -92,6 +92,15 @@
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = True
 
+napoleon_custom_sections = [
```

Comment:
Thanks for adding the comment.

Is there a way to somehow make this config apply only to the SlowMo doc? Otherwise it seems a bit brittle, for instance if someone uses the "Debugging Parameters" string in another completely unrelated docstring, this may break their doc, right?

Reply:
I can possibly change to SlowMoDDP Debugging Parameters instead to quickly resolve this issue but I guess it's not a very clean solution

Reply:
Apparently you can pass a list of tuples but I don't fully understand what this achieves, can you just play a bit with it to see if it might help?

Reply:
So the list of tuples implies that it treats the first parameter of the tuple as an alias for the second parameter (a section) just with a different name. I don't see a way so far to resolve this after looking into it a bit. I'm thinking of renaming "Debugging Parameters" to "SlowMo Debugging Parameters" to remove the chance of future errors. Do you think that might be okay to go ahead with?

Reply:
Will also wait for @blefaudeux's thoughts on this before going ahead with anything

Reply:
I had a closer look, and actually I don't think the current config does what you want it to do (compare the formatting of the `Parameters` section vs. `SlowMo Parameters`). I messed a bit with it but couldn't figure out how to achieve what you want (not sure if that's possible).

Reply:
So it needs an upgrade of sphinx to 3.5.0 before it works. This [PR](https://github.com/sphinx-doc/sphinx/pull/8658) introduced the feature that makes the formatting the same

Reply:
> So it needs an upgrade of sphinx to 3.5.0 before it works. This [PR](https://github.com/sphinx-doc/sphinx/pull/8658) introduced the feature that makes the formatting the same

Ah, thanks, working now! :)

Now I better understand what this does. I actually think it's fine to keep "Debugging Parameters" as it may prove useful in other parts of fairscale in the future.

However, it seems that sphinx (at least 3.5.0) doesn't like the last section due to the parentheses. One suggestion would be to rename it into ""Optional Parameters for Advanced Users".

Reply:
Thanks I had renamed it to "Parameters for Advanced Users" earlier (still had to upstream that). But definitely, adding Optional makes it even better

Reply:
Actually on second thoughts, most of the parameters above are Optional. So, I'm dropping the Optional part. The default values of the parameters (in addition to the name that it's for "Advanced Users" should make it clear to the user that these parameters are optional